projects/demo/src/modules/components/app-bar/examples/3/index.html

```diff
@@ -47,7 +47,9 @@ <h3>Customization</h3>
         tuiIconButton
         tuiSlot="right"
         type="button"
-    ></button>
```

Comment:
Use title here, it has no text on purpose

Reply:
fixed